ui/src/app/views/workspace-edit/component.ts

```diff
@@ -101,7 +101,7 @@ export class WorkspaceEditComponent implements OnInit {
   workspaceCreationConflictError = false;
   workspaceUpdateError = false;
   workspaceUpdateConflictError = false;
-  private accessLevel: WorkspaceAccessLevel;
+  accessLevel: WorkspaceAccessLevel;
```

Comment:
Sorry! I totally let this discussion get derailed, was this where you landed? Best option?